test/e2e/tas/pod_group_test.go

```diff
@@ -94,11 +97,16 @@ var _ = ginkgo.Describe("TopologyAwareScheduling for Pod group", func() {
 				Queue("test-queue").
 				Request(extraResource, "1").
 				Limit(extraResource, "1").
-				Annotation(v1alpha1.PodSetRequiredTopologyAnnotation, "cloud.provider.com/topology-rack")
+				Annotation(kueuealpha.PodSetRequiredTopologyAnnotation, "cloud.provider.com/topology-block")
 			podGroup := basePod.MakeIndexedGroup(numPods)
 
 			for _, pod := range podGroup {
 				gomega.Expect(k8sClient.Create(ctx, pod)).Should(gomega.Succeed())
+				gomega.Expect(pod.Spec.SchedulingGates).To(gomega.ContainElements(
+					corev1.PodSchedulingGate{Name: podcontroller.SchedulingGateName},
+					corev1.PodSchedulingGate{Name: kueuealpha.TopologySchedulingGate},
+				))
+				gomega.Expect(pod.Labels).To(gomega.HaveKeyWithValue(kueuealpha.TASLabel, "true"))
```

Comment:
TBH, I testing this is not the purpose of the test, especially `podcontroller.SchedulingGateName`. Moreover, it can be problematic and flaky if the workload is admitted quickly, and thus the scheduling gates are removed quickly, you may not be able to assert they were added in the first place. I know this typically takes like half second, so the test would pass, but we don't write asserts which have  potential to flake.

Reply:
Good point, fixed